kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -4,6 +4,18 @@
 
 use rustc_ast::{AttrKind, Attribute, LitKind, MetaItem};
 
+/// An enum for possible errors with attribute extraction
+pub enum AttrError {
```

Comment:
Should this be `KaniToolAttrError` or `KaniAttrError`? I thought these came from `rustc_ast` but then found this enum.

Reply:
I removed this attribute.